tests/unit/commands/test_code.py

```
@@ -0,0 +1,137 @@
+        opened_path = Path(called_with_path[0])
+        assert opened_path.name != "."
+
+    def test_code_command_subprocess_error(
+        self, cli_runner: CliRunner, monkeypatch
+    ) -> None:
+        """Test that code command handles subprocess errors gracefully."""
+        import shutil
+        import subprocess
+
+        # Mock shutil.which to return a path (command found)
+        monkeypatch.setattr(shutil, "which", lambda cmd: "/usr/local/bin/code")
```

Ruff (0.14.10)

Unused lambda argument: `cmd`

(ARG005)